odnoklassniki-android-sdk/src/main/java/ru/ok/android/sdk/Odnoklassniki.java

```diff
@@ -445,4 +504,48 @@ private void clearCookiesOld() {
         CookieManager cookieManager = CookieManager.getInstance();
         cookieManager.removeAllCookie();
     }
+
+
+    /**
+     * Inner class to store
+     */
+    private class DeferredData {
+        private boolean isDeferredDataError;
+
+        private String deferredErrorCode;
```

Comment:
Having deferred in each variable and getters makes names too long with no reason (the bean already contains the prefix), thus better rename